mn_manager/src/wattrex_mn_manager/mn_manager_node.py

```diff
@@ -0,0 +1,150 @@
+
+#######################    SYSTEM ABSTRACTION IMPORTS    #######################
+from system_logger_tool import sys_log_logger_get_module_logger, SysLogLoggerC, Logger
+
+#######################       LOGGER CONFIGURATION       #######################
+if __name__ == '__main__':
+    cycler_logger = SysLogLoggerC(file_log_levels='./log_config.yaml')
+log: Logger = sys_log_logger_get_module_logger(__name__)
+
+#######################          PROJECT IMPORTS         #######################
+from system_shared_tool import SysShdIpcChanC, SysShdNodeC
+from wattrex_battery_cycler_datatypes.comm_data import (CommDataCuC,CommDataRegisterTypeE,
+                                                        CommDataHeartbeatC, CommDataDeviceC,
+                                                        CommDataMnCmdDataC, CommDataMnCmdTypeE)
+
+#######################          MODULE IMPORTS          #######################
+from .mn_broker_client import BrokerClientC
+from .mn_db_facade import DbFacadeC
+
+#######################              ENUMS               #######################
+MN_REQS_CHAN_NAME = 'mn_reqs'
+MN_DATA_CHAN_NAME = 'mn_data'
+
+#######################             CLASSES              #######################
+
+class MnManagerNodeC(SysShdNodeC): # pylint: disable=abstract-method
+    '''
+    Cu Manager Class to instanciate a CU Manager Node
+    '''
+    def __init__(self, working_flag : threading.Event, cycle_period : int) -> None:
+        '''
+        Initialize the node.
+        '''
+        super().__init__(name='cu_manager_node', cycle_period=cycle_period,
+                working_flag=working_flag)
+
+        self.mn_req_chan : SysShdIpcChanC = SysShdIpcChanC(name=MN_REQS_CHAN_NAME,
+                                                            max_message_size=350,
+                                                            max_msg=350)
+        self.mn_data_chan : SysShdIpcChanC = SysShdIpcChanC(name=MN_DATA_CHAN_NAME,
+                                                            max_message_size=350,
+                                                            max_msg=350)
+        self.db_facha : DbFacadeC = DbFacadeC()
+        avail_cus : List[int] = self.db_facha.get_available_cus()
+        log.info(f"Already connected devices: {avail_cus}")
+
+        self.client_mqtt : BrokerClientC = BrokerClientC(error_callback=self.error_cb,
+                                                         register_cb=self.register_cb,
+                                                         heartbeat_cb=self.heartbeat_cb,
+                                                         inform_dev_cb=self.detect_devices_cb,
+                                                         avail_cus=avail_cus)
+
+    def error_cb(self, data) -> None:
+        '''Error callback for Broker.
+
+        Args:
+            data ([type]): [description]
+        '''
+        log.critical(f'Error in Broker Client: {data}')
```

Comment:
It not update status. Should we do that?

Reply:
What status?

Reply:
SysShdNodeStatusE

Reply:
Done. Set status to COMM_ERROR